server/src/main/java/org/elasticsearch/action/admin/indices/mapping/get/GetFieldMappingsRequest.java

```diff
@@ -59,26 +57,17 @@ public GetFieldMappingsRequest(StreamInput in) throws IOException {
             if (types != Strings.EMPTY_ARRAY) {
                 throw new IllegalArgumentException("Expected empty type array but received [" + Arrays.toString(types) + "]");
             }
+
         }
         indicesOptions = IndicesOptions.readIndicesOptions(in);
-        local = in.readBoolean();
+        // Consume the deprecated local parameter
+        if (in.getVersion().before(Version.V_8_0_0)) {
+            in.readBoolean();
+        }
```

Comment:
This is not necessary if we can be sure that the request never goes across wire.